CHANGELOG.md

```
@@ -9,7 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added setup workflow for GitHub Copilot.
   - Switch the workflow to use Linux.
-  - Skip verify gitversion after install due to problem with how Copilot switches branch directly after checkout, before setup is finsihed.
+  - Skip verify GitVersion after install due to problem with how Copilot switches branch directly after checkout, before setup is finsihed.
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 80; Actual: 139

(MD013, line-length)